plugin/pkg/scheduler/algorithm/listers.go

```diff
@@ -184,3 +185,42 @@ func (f FakeReplicaSetLister) GetPodReplicaSets(pod *v1.Pod) (rss []*extensions.
 
 	return
 }
+
+// StatefulSetLister interface represents anything that can produce a list of StatefulSet; the list is consumed by a scheduler.
+type StatefulSetLister interface {
+	// Gets the StatefulSet for the given pod.
+	GetPodStatefulSets(*v1.Pod) ([]*apps.StatefulSet, error)
```

Comment:
Sorry I should have deleted this comment I looked in the other code and it does not.